lib/hunter/status.ex

```diff
@@ -82,7 +84,7 @@ defmodule Hunter.Status do
     * `media_ids` - [Array<Integer>]
 
   """
-  @spec create_status(Hunter.Client.t, String.t, non_neg_integer, [non_neg_integer]) :: Hunter.Status.t
+  @spec create_status(Hunter.Client.t, String.t, status_id, [status_id]) :: Hunter.Status.t
```

Comment:
If you want to be explicit, which is fine, why are you assuming that `media_ids` it's the same type as `status_id`?